salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -7,7 +7,9 @@ edx:
         - DeviceIndex: 0
           AssociatePublicIpAddress: True
           SubnetId: {{ subnetids[id_num % num_subnets] }}
-          SecurityGroupId: {{ securitygroupid }}
+          SecurityGroupId: |
+            {{ securitygroupids.edxapp, securitygroupids.edx-worker, securitygroupids.default,
```

Comment:
This should be formatted as a list, and I don't believe that calling the multiple values in a single Jinja reference will work...